### DISCUSSION TITLE
Citation key format Windows

### DISCUSSION THREAD

**Discussion by kydmar** [retorquere/zotero-better-bibtex#3003] [Q&A] [answered]
I need citation key of the format Author year (like Smith2022). I copied to the new laptop (Windows 11) the setting from the old one, also 64 bit. Bibtex now produces citation keys with the name and year, but  the name is capitalized only if there is a red button indicating a link to the paper file on Zotero. Can this be fixed?
Thank you
kydmar

**Comment by retorquere** [comment, marked as answer]
If you want something fixed, open an issue, not a discussion.

**Comment by kydmar** [comment]
Opening an issue seems to be more complicated than just quitting zotero.
Too bad.

**Reply by retorquere** [reply]
suit yourself.
